var/spack/repos/builtin/packages/expat/package.py

```diff
@@ -102,19 +105,36 @@ class Expat(AutotoolsPackage):
     # `~libbsd`.
     variant(
         "libbsd",
-        default=sys.platform != "darwin",
+        default=sys.platform != "darwin" and sys.platform != "win32",
         description="Use libbsd (for high quality randomness)",
     )
 
+    variant("shared", default=True, description="Build expat as shared if true, static if false")
```

Comment:
I think this should be removed: if I understand correctly, the autotools build makes shared and static libs, and this variant does nothing.

If you think the cmake-based build needs this option, I recommend only allowing the variant when `build_system=cmake`.

Reply:
I think removing the option is leaving some utility out (someone may want static over shared some day) so I'll leave it for CMake only. Pushed.